smart-contracts/contracts/UnlockTestV3.sol

```diff
@@ -0,0 +1,87 @@
+pragma solidity 0.4.24;
+
+/**
+ * @title Example upgrade of the Unlock contract
+ * @author Julien Genestoux (unlock-protocol.com)
+ * @author HardlyDifficult
+ * 
+ * This is a copy paste of the current Unlock contract, with comments removed.
+ * Then a few small changes have been made, higlighted with comments.
+ */
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./Lock.sol";
+
+
+contract UnlockTestV3 is Ownable {
```

Comment:
So I guess if we were *not* using `is Unlock`, we can then re-inplement it all?

Reply:
Yes. Either approach but if using this style be careful not to change the order data appears.

Reply:
Yup. Makes sense!